### PR TITLE
Feature/spoty

### PR DESCRIPTION
**Pull request by AsierDev**
My RC spotify app. Should be functional. 
Left to do => change the api from ajax to fetch and transform code to js6.

### CODE REVIEW

**Comment by AsierDev** [comment]
### A few new features for this PR

- I've implemented jasmine for testing. It passes all tests. 
- Change code to js6
- Move API from ajax to fetch
- Polished some details

File: spotify-app/spotify-rc-asier/scripts/main.js [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+    // ------- submit form and get artists, pass them to the api and show them on html -------//
+
+    $('form').submit(function (e) {
+        e.preventDefault();
+        const query = $('input').val()
+
+        $('form').trigger('reset')
+        $("#listArtists").show()
+        $("#listAlbums, #listSongs, #backToArtists, #error").hide()
```
Comment (VGamezz19) [inline spotify-app/spotify-rc-asier/scripts/main.js]: hey! you can use other thing.
```javascript
const $mainContainer = $('.container-fluid')
$mainContainer.childer().hiden()
```

File: spotify-app/spotify-rc-asier/scripts/spotify-api-1.0.0.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+
+let spotifyApi;
+(function () {
+    "use strict";
+
+
+
+    function call(url, token, handleSuccess, handleError, timeout) {
+
```
Comment (VGamezz19) [inline spotify-app/spotify-rc-asier/scripts/spotify-api-1.0.0.js]: Same as PR #52, your fetch request didn't have any control with timeout ⚠️

File: spotify-app/spotify-rc-asier/scripts/main.js (hunk 1 above)
Comment (AsierDev) [reply spotify-app/spotify-rc-asier/scripts/main.js]: I could, but 'form' is a 'main' child too... So I'd had to specify all children but 'form' to hide

**Pull request merged**